app/views/groups/index.html.erb

```diff
@@ -47,6 +47,10 @@
   <%= raw t('groups.index.instructions', {
       icon: '<i class="fa fa-plus-circle"></i>'
     }) %>
+  <br /><br />
```

Comment:
Using `<br>` should be sufficient. Though I would either wrap the lines before that in a `<p>` or replace `<br /><br />` with `<div class="spacer"></div>`